lib/tessel/deployment/rust.js

```diff
@@ -86,6 +86,10 @@ exportables.remoteRustCompilation = (opts) => {
     var buffers = [];
 
     // Parse out the components of the remote rust compilation server
+    if (!opts.rustCC.startsWith('http://')) {
```

Comment:
You may want to also check that it starts with `https://` in case someone deploys their own cross compilation server somewhere behind SSL (or if we do that...).